### PR TITLE
adapter,externals setup

### PR DESCRIPTION
**Pull request by celiibrendan**

### CODE REVIEW

File: python/microns-materialization-config/microns_materialization_config/__init__.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
+"""
+Configuration package/module for microns-MATERIALIZATION.
```
Comment (spapa013) [inline python/microns-materialization-config/microns_materialization_config/__init__.py:2]: lowercase

File: python/microns-materialization-config/microns_materialization_config/externals.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    return {
+        'protocol': 'file',
+        'location': str(path),
+        'stage': str(path)
+    }
+
+
+#h01 materialization
+h01_materialization_external_meshes_path = Path() / '/mnt' / 'dj-stor01' / 'microns'/ 'h01' / 'meshes'
+h01_materialization = {
```
Comment (spapa013) [inline python/microns-materialization-config/microns_materialization_config/externals.py:13]: can we combine as such:
```python
h01_materialization = {
    'h01_meshes': make_store_dict(Path() / '/mnt' / 'dj-stor01' / 'microns'/ 'h01' / 'meshes'),

    }

File: python/microns-materialization/h01_materialization/h01_materialization.py [hunk 3]
```diff
@@ -9,11 +9,18 @@
...
     from tqdm import tqdm
 
+import microns_materialization_config as config
+schema_name = 'microns_h01_materialization'
+
+config.register_adapters(schema_name)
+config.register_externals(schema_name)
+
 # Schema creation
-schema = dj.schema('microns_h01_materialization', create_tables=True)
```
Comment (spapa013) [inline python/microns-materialization/h01_materialization/h01_materialization.py:18]: change to `create_tables=False`

File: python/microns-materialization/minnie_materialization/minnie65_materialization.py [hunk 4]
```diff
@@ -16,7 +16,14 @@
...
 # Schema creation
-schema = dj.schema('microns_minnie65_materialization', create_tables=True)
+import microns_materialization_config as config
+schema_name = 'microns_minnie65_materialization'
+
+config.register_adapters(schema_name)
+config.register_externals(schema_name)
+
+# Schema creation
+schema = dj.schema(schema_name, create_tables=True)
```
Comment (spapa013) [inline python/microns-materialization/minnie_materialization/minnie65_materialization.py:26]: change to `create_tables=False`

**Review by spapa013 — CHANGES_REQUESTED** [review]

**Review by spapa013 — APPROVED** [review]

**Review by Cpapa97 — APPROVED** [review]
Looks good.

**Pull request merged**